thirdparty/github.go

```diff
@@ -722,3 +722,11 @@ func buildPatchURL(gp *patch.GithubPatch) string {
 
 	return url.String()
 }
+
+func GithubPRMerge(title string, message string, sha string, mergeMethod string) error {
```

Comment:
You can simplify Go signatures by referring to a repeated type only once, e.g., `(title, message, sha, mergeMethod string)`. However, with this many arguments, it may be better to pass a struct, as callers can easily confuse between the arguments.

Reply:
Done.